instrumentation/aws-sdk/aws-sdk-2.2/testing/src/main/groovy/io/opentelemetry/instrumentation/awssdk/v2_2/AbstractAws2ClientTest.groovy

```diff
@@ -72,15 +72,15 @@ class Aws2ClientCoreTest extends InstrumentationTestRunner {
     }
   }
 
-  def "send #operation request with builder {#builder.class.getName()} mocked response"() {
+  abstract void configureSdkClient(SdkClientBuilder builder)
+
+  def "send #operation request with builder #builder.class.getName() mocked response"() {
     setup:
+    configureSdkClient(builder)
```

Comment:
Are you sure you want to call this on every method? Not in `setupSpec`?

Reply:
I might be misreading, but I think it's not possible here since the builder is actually created in the where clause of the spec, it's one AWS SDK but there are actually many different types of builders